library/ziti.c

```diff
@@ -1588,6 +1588,29 @@ void ziti_queue_work(ziti_context ztx, ztx_work_f w, void *data) {
     uv_async_send(&ztx->w_async);
 }
 
+static void copy_oidc(ziti_context ztx, const ziti_jwt_signer *oidc) {
+    if (oidc == NULL) return;
+    if (oidc->provider_url == NULL) {
+        ZITI_LOG(ERROR, "invalid OIDC config `externalAuthUrl` is missing");
+        return;
+    }
+    if (oidc->client_id == NULL) {
+        ZITI_LOG(ERROR, "invalid OIDC config `clientId` is missing");
+        return;
+    }
+
+    ztx->config.id.oidc = calloc(1, sizeof(*oidc));
```

Comment:
do you clean this (and the strdups) up, or it it leaky?

Reply:
ztx->config is cleaned up when it is closed